addons/base_import_module/tests/test_cloc.py

```diff
@@ -50,6 +50,15 @@ def test_count_qweb_imported_module(self):
             'state': 'installed',
             'summary': 'Test imported module for cloc',
         })
+        # Studio module does not exist at this stage, so we simulate it
```

Comment:
the ir.module.module has a unique constraint. Maybe make this create in a try/except in case we run this test in a db where it exists?

Reply:
The installation of web_studio does not create the record, only at the usage. Try except won't be enough if the unique constraint is violate since the cursor will be broken. Let's try to prevent it then